antwar.config.js

```diff
@@ -67,6 +67,8 @@ module.exports = {
           /^\.\/.*\.md$/
         );
       }, {
+        'code-splitting-import': '/guides/code-splitting-import',
```

Comment:
Since the `import()` part starts right on the top (after a short introduction), I refrained from adding an anchor.